exec/cpg.c

```diff
@@ -194,6 +194,12 @@ struct join_list_entry {
 	mar_cpg_name_t group_name;
 };
 
+struct join_list_confchg_data {
+	mar_cpg_name_t cpg_group;
+	mar_cpg_address_t join_list[CPG_MEMBERS_MAX];
+	int join_list_entries;
+};
+
```

Comment:
instead of defining this here, we could also make `do_proc_join` return/alloc/fill a `mar_cpg_address_t` and make the respective callers call `notify_lib_joinlist`, then this would only be needed in `joinlist_inform_clients` similar to how it is done in `downlist_inform_clients`?